assembly/runtime/collections/persistentDeque.ts

```diff
@@ -71,21 +100,51 @@ export class PersistentDeque<T> {
   }
 
   /**
-  * @param index The index to check.
-  * @returns True if the given index is within the range of the deque indices.
-  */
+   * Checks whether the index is within the range of the deque indices
+   *
+   * ```ts
+   * let dq = PersistentDeque<string>("q")
+   *
+   * dq.containsIndex(0) // false
+   * dq.pushFront("hello world")
+   * dq.containsIndex(0) // true
+   * ```
+   *
+   * @param index The index to check.
+   * @returns True if the given index is within the range of the deque indices.
+   */
   containsIndex(index: i32): bool {
     return index >= 0 && index < this.length;
   }
 
   /**
-  * @returns The length of the deque.
-  */
+   * Returns the length of the deque
+   *
+   * ```ts
+   * let dq = PersistentDeque<string>("q")
+   *
+   * dq.length // 0
+   * dq.pushFront("hello world")
+   * dq.length // 1
+   * ```
+   *
+   * @returns The length of the deque.
+   */
   get length(): i32 {
     return this.backIndex - this.frontIndex + 1;
   }
 
   /**
+   * Returns a boolean indicating whether the deque is empty
```

Comment:
Again seems redundant.

Reply:
yep, same as above. what's the right way to handle this?  bc this happens all over the place

Reply:
I think change to sentences like: "Indicates whether the deque is empty".